packages/communication-react/package.json

```diff
@@ -74,12 +85,14 @@
     "build:cjs": "rollup -c --silent",
     "build:esm": "npm run _if-preprocess && rushx copy-original && rushx preprocess && rushx copy-preprocess  && tspc -project tsconfig.preprocess.json || (if-env COMMUNICATION_REACT_FLAVOR=beta && tspc)",
     "build:watch": "",
+    "build:api": "rushx _api-extractor:complete:by-flavor",
     "clean": "rimraf dist && rimraf ../**/preprocessed && rimraf ../**/preprocess-dist",
     "test": "",
     "test:ci-coverage": "",
     "test:coverage": "",
     "check-deps": "node scripts/verifyDependencies",
     "api-extractor": "rushx _by-flavor \"rushx build:esm && rushx _api-extractor:by-flavor\"",
```

Comment:
Looking at the the line above, api-extractor should be similar to the line above, so `rushx _by-flavor` would be also in the command of your new command

Reply:
Thanks for the comment I added it as well!